spec/models/concerns/date_helpers_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe DateHelpers do
+  let(:test_class) do
+    Class.new { include DateHelpers }
+  end
+
+  subject { test_class.new }
+
+  context 'valid_date?' do
+    it 'method should be defined' do
+      expect(subject.private_methods.include? :valid_date?).to eq true
```

Comment:
just to be sure the module is written correctly and the right module was imported.